Lib/test/test_os.py

```diff
@@ -179,6 +179,13 @@ def test_tmpnam(self):
             else:
                 self.check_tempfile(name)
 
+    @unittest.skipUnless(sys.platform == 'win32', 'tests windows behaviour')
```

Comment:
Do WinCE or OS/2 support such path syntax?

Reply:
I don't actually have access to either of those environments -- is there an easy way I can check?